upgrade_test.py

```diff
@@ -66,6 +66,35 @@ def check_reload_systemd_config(node):
             raise Exception("Systemd config is changed, but not reload automatically")
 
 
+def backup_conf(node):
+    if node.is_rhel_like():
+        node.remoter.run(
+            r'for conf in $( rpm -qc $(rpm -qa | grep scylla) | grep -v contains ) '
+            r'/etc/systemd/system/{var-lib-scylla,var-lib-systemd-coredump}.mount; '
```

Comment:
it seems like run `sudo cp -v $conf $conf.autobackup` unconditionally on .mount files, but are these files generated in the test? (normally it means scylla_setup is executed).
If not, I think this cp will fail.

Reply:
The command is executed in a living cluster, scylla_setup has been executed in prepare stage, so the mount configs exist.